source/uwp/SharedVisualizer/ViewModel/KeyboardPressTimeCounter.cs

```diff
@@ -56,7 +61,18 @@ public void CheckIfSendIsNecessary(Object stateInfo)
 
         private async Task SendShowErrorsUpdateAsync()
         {
+#if USE_WINUI3
+            var dispatcher = Microsoft.UI.Dispatching.DispatcherQueue.GetForCurrentThread();
+            if (dispatcher == null)
+            { 
+                var queueController = Microsoft.UI.Dispatching.DispatcherQueueController.CreateOnCurrentThread();
```

Comment:
could we just take in the dispatcher queue as a param to this object or method?

Reply:
Getting it to this ViewModel object means passing it through about four other constructors and a bunch of methods, and especially messy with the different types between uwp+winui3. It's a large enough change that I'd rather not, unless doing it this way is particularly bad.